packages/ui/label/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from 'label'
+export * from 'label'
```

Comment:
These should be `@littlespoon/label`:

```suggestion
export { default } from '@littlespoon/label'
export * from '@littlespoon/label'
```

Reply:
Updated